File: list_1/Multimetro.java
import java.util.Scanner;
import java.lang.Math;

public class Multimetro {

    public static void main(String[] args){
        double media = 0, media2;
        double variancia = 0,desvio;
        double[] vetor = {3.2, 4.8, 2.5, 6.1, 3.9, 5.6, 2.2, 4.3, 6.8, 3.7};
        double[] dif = new double[10];
        for(int i = 0;i < 10;i++){
            media += vetor[i];
        }
        media /= 10;

        for(int i = 0;i < 10;i++){
            dif[i] = vetor[i] - media;
            dif[i] *= dif[i];
            variancia += dif[i];
        }
        variancia /= 10;
        desvio = Math.sqrt(variancia);
        if(desvio > media){
            media2 = (media / 100) * 10;
            double aux = desvio - media;
            if(aux >= media2) {
                System.out.println("Esse multimetro esta com problemas!");
                System.exit(0);
            }
        }
        System.out.printf("Desvio: %f\n", desvio);
        System.out.printf("Variancia: %f\n", variancia);
        System.out.printf("Media: %f\n", media);


    }
}

File: list_2/Multimetro.py
import math
media = media2 = variancia = desvio = 0
vetor = [3.2, 4.8, 2.5, 6.1, 3.9, 5.6, 2.2, 4.3, 6.8, 3.7]
dif = [0] * 10

for numero in range(0, 10):
    media += vetor[numero]
media /= 10

for num in range(0, 10):
    dif[num] = vetor[num] - media
    dif[num] *= dif[num]
    variancia += dif[num]

variancia /= 10
desvio = math.sqrt(variancia)
if desvio > media:
    media2 = (media / 100) * 10
    aux = desvio - media
    if aux >= media2:
        print("Esse multimetro esta com problemas!!!")
        exit(-1)
print(f"Desvio: {desvio}")
print(f"Variancia: {variancia}")
print(f"Media: {media}")
